src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -164,4 +165,34 @@ describe('RadioButtonGroup', () => {
       assertStyleMatch(customStyleObject, wrapper.find(StyledLegendContainer));
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = mount(
+      <RadioButtonGroup
+        name='radio' label='Group Label'
+        required
+      >
+        <RadioButton
+          label='off'
+          value='test'
+        />
+        <RadioButton
+          label='on'
+          value='test'
+        />
+      </RadioButtonGroup>
+    );
+
+    const inputs = wrapper.find('input');
+    inputs.forEach((input) => {
+      expect(input.prop('required')).toBe(true);
+    });
+
+    const fieldset = wrapper.find(Fieldset);
+    expect(fieldset.prop('isRequired')).toBe(true);
+
+    const labels = wrapper.find(Label);
+    expect(labels.at(0).prop('isRequired')).toBe(false);
+    expect(labels.at(1).prop('isRequired')).toBe(false);
+  });
```

Comment:
three tests in one case

Reply:
Updated test description because the setup is the same.